### PR TITLE
change btn addCard + add animation

### PR DESCRIPTION
**Pull request by Andry-art**

### CODE REVIEW

File: packages/App/src/balance/debits/Debits.tsx [hunk 1]
```diff
@@ -60,6 +65,21 @@ const Debits: FC<Props> = ({navigation}) => {
   const deleteError = useSelector(deleteDebitError);
   const debitsError = useSelector(getErrorDebits);
 
+  const debToYouInitRotate = useSharedValue(0)
+  const yourDebInitRotate = useSharedValue(0)
+
+  const debToYouAnimated = useAnimatedStyle(() => {
```
Comment (sergej-prilutskiy) [inline packages/App/src/balance/debits/Debits.tsx]: Let's rename it to `debtsToYouAnimatedStyle`

File: packages/App/src/balance/debits/Debits.tsx [hunk 2]
```diff
@@ -60,6 +65,21 @@ const Debits: FC<Props> = ({navigation}) => {
...
+    return {
+      transform: [{ rotate: `${debToYouInitRotate.value}deg` }],
+    };
+  });
+
+  const yourDebAnimated = useAnimatedStyle(() => {
```
Comment (sergej-prilutskiy) [inline packages/App/src/balance/debits/Debits.tsx]: Same here

File: packages/App/src/balance/debits/Debits.tsx [hunk 3]
```diff
@@ -75,11 +95,26 @@ const Debits: FC<Props> = ({navigation}) => {
   const DebitsToYou = useCallback(() => {
     setDebitsVisible(prev => !prev);
     setMyDebitsVisible(false);
+   if (debToYouInitRotate.value === 0) {
```
Comment (sergej-prilutskiy) [inline packages/App/src/balance/debits/Debits.tsx]: It's better to change values in some effect depending on state

File: packages/App/src/balance/wallet/WalletItem.tsx [hunk 4]
```diff
@@ -73,6 +92,13 @@ const styles = StyleSheet.create({
     backgroundColor: '#FFFFFF',
   },
 
+  plus: {
+    fontSize: 60,
+    color: 'white',
+  },
+
```
Comment (sergej-prilutskiy) [inline packages/App/src/balance/wallet/WalletItem.tsx]: Why blank spaces?

File: packages/App/src/store/saga/walletItemsRequest.tsx [hunk 5]
```diff
@@ -33,6 +33,15 @@ export function* getWalletItems(): Generator {
     )) as Array<WalletInfo>;
 
     if (response) {
+      response.push( {
```
Comment (sergej-prilutskiy) [inline packages/App/src/store/saga/walletItemsRequest.tsx]: It'll be better not to have this logic in sagas. Sagas and redux are for data only. This special item is part of UI logic, so it should be resolved in UI components, not here

**Review by sergej-prilutskiy — CHANGES_REQUESTED** [review]

File: packages/App/src/store/saga/walletItemsRequest.tsx [hunk 6]
```diff
@@ -33,6 +33,7 @@ export function* getWalletItems(): Generator {
     )) as Array<WalletInfo>;
 
     if (response) {
+      response;
```
Comment (sergej-prilutskiy) [inline packages/App/src/store/saga/walletItemsRequest.tsx]: ?)

**Review by sergej-prilutskiy — APPROVED** [review]
Let's fix that minor thing and the PR can be merged

File: packages/App/src/store/saga/walletItemsRequest.tsx (hunk 6 above)
Comment (Andry-art) [reply packages/App/src/store/saga/walletItemsRequest.tsx]: yep)

Comment (Andry-art) [reply packages/App/src/store/saga/walletItemsRequest.tsx]: I moved logic from here to UI)and there added card-button, that add new card)

Comment (sergej-prilutskiy) [reply packages/App/src/store/saga/walletItemsRequest.tsx]: That isn't. You are not returning `response`

Comment (Andry-art) [reply packages/App/src/store/saga/walletItemsRequest.tsx]: ahh right) there is no need to return actually
I deleted it, and make commit

**Pull request merged**